jobs/pingcap/tiflow/latest/ghpr_verify.groovy

```diff
@@ -0,0 +1,79 @@
+// REF: https://<your-jenkins-server>/plugin/job-dsl/api-viewer/index.html
+// For trunk and latest release branches.
+pipelineJob('pingcap/tiflow/ghpr_verify') {
+    logRotator {
+        daysToKeep(30)
+    }
+    parameters {
+        stringParam("ghprbActualCommit")
+        stringParam("ghprbPullId")
+    }
+    properties {
+        // priority(0) // 0 fast than 1
+        githubProjectUrl("https://github.com/pingcap/tiflow")
+        pipelineTriggers {
+            triggers {
+                ghprbTrigger {
+                    cron('H/5 * * * *')
+                    gitHubAuthId('') // using the default only one.
+
+                    triggerPhrase('.*/(run-verify.*)|(/merge)')
+                    onlyTriggerPhrase(false)
```

Comment:
it's will be triggered automatic when PR refreshed or created, I think it's no need to recall it when user comment with '/merge'

Reply:
Please check rest parts.

Reply:
This is a current user habit, and this behavior change requires advance notice. This time only migration will be performed